tests/cypress/e2e/components/media.cy.ts

```diff
@@ -74,21 +76,21 @@ const mediaType = [
     tagRelated: 'model-viewer',
     type: '3d',
   },
+  // {
+  //   url: '/rmrk/gallery/9684463-c8205518d881699b3e-%E2%9D%84%EF%B8%8F-01_FROM_SINGULARITY-0000000000000005',
+  //   title: '01 fr.om - singularity',
+  //   description: '',
+  //   collection: 'glaciers mood',
+  //   creator: 'fr.om',
+  //   tagRelated: 'img',
+  //   type: 'audio',
```

Comment:
is the audio facing the same problem if we change to shorter/smaller audio?

Reply:
unfortunately yes, i've also tried /rmrk/gallery/14157116-64274F293D519F050C-F4TVXP2MGJ7FA-UNSUPPORTED_IMAGE_FORMAT_1-0000000000000001 which is under one second and only a few kb in size